Makefile

```diff
@@ -274,6 +274,10 @@ MOD_JDK_SECURITY_JGSS_FZ_FILES = $(MOD_JDK_SECURITY_JGSS_DIR)/__marker_for_make_
 MOD_JDK_XML_DOM_FZ_FILES = $(MOD_JDK_XML_DOM_DIR)/__marker_for_make__
 MOD_JDK_ZIPFS_FZ_FILES = $(MOD_JDK_ZIPFS_DIR)/__marker_for_make__
 
+MOD_FLANG_DIR = $(BUILD_DIR)/modules/flang
+MOD_FLANG_FZ_FILES = $(MOD_FLANG_DIR)/__marker_for_make__
+MOD_FLANG = $(MOD_FLANG_DIR).fum
```

Comment:
Can you rename these as `MOD_FZ_CMD` and `MOD_FZ_CMD_DIR`, and use `$(BUILD_DIR)/modules/fz_cmd`.